# Operating cost

This section computes operating cost for the example airplane. From the perspective of an airline, the operating costs can be divided into direct operating cost (DOC) and indirect operating cost (IOC). The DOC consists of costs associated with flight operations such as fuel, oil, crew, maintenance, depreciation, etc. The IOC comprises of costs for running an airline such as ground facilities, equipment, administrative and overhead charges. Note that it is very difficult to estimate IOC since it is largely based on how an airline is managed. Refer to section 18.7 in Raymer for more details about airline economics.

However, the operating cost in this demonstration is based on the assumption that the owner itself is a pilot i.e. the airplane is bought for personal use only. The cost model uses a simple bookeeping and tracking of several costs associated with operating a GA airplane. The primary inputs are flight hours per year and cost of fuel, which are then used to compute various elements of operating cost. Refer to section 2.3 in Gudmundsson for more details. Note that the annual operating cost is estimated.

## Maintenance cost

The maintenance cost is based on the maintenance work hours in a year and the hourly wage for the mechanic who does maintenance. The maintenance work hours is based on several factors such as flight hours flown in a year, refer to section 2.3.1 in Gudmundsson's book for more details. The wage for a certified mechanic is set to $40/hr based on typical wages found online. The number of flight hours per year is set to 300 hours based on Gudmundsson's suggested range of values. Below code block computes maintenance cost for the example airplane.

In [1]:
R_mechanic = 40 # $/hr, A&P mechanic rate
HF = 300 # hours flown/year - personal use only

F1 = 0 # maintained by A&P mechanic
F2 = 0 # easy access for engine
F3 = 0.02 # retractable landing gear
F4 = 0.02 # VFR
F5 = 0.04 # IFr
F6 = 0.01 # integral tanks
F7 = 0 # simple flaps
F8 = 0 # 14 CFR Part 23

# maintenance to flight hour ratio
F_mf = 0.3 + F1 + F2 + F3 + F4 + F5 + F6 + F7 + F8

C_maintenance = F_mf * R_mechanic * HF # $/year, maintenance

print(f"Maintenance cost per year: ${int(C_maintenance):,}")

Maintenance cost per year: $4,680


## Fuel cost

The fuel cost is computed based on the specific fuel consumption, flight hours flown in a year, average power during cruise, and cost for a gallon of fuel. The specific fuel consumption can be obtained from the engine selected, however, it is set to 0.4 lbs/hp/hr in this section. The flight hours in a year is already determined in maintenance cost section and the average power during cruise is obtained from trade studies. Finally, the average cost for a gallon of fuel (100 Avgas) is set to $6.4. Below code block computes annual fuel cost for the example airplane:

In [2]:
Pcruise = 530 # hp, for both engine combined
fuel_density = 6 # lbs/gallon
Cbhp = 0.4 # lbs/hp/hr
fuel_cost_gallon = 6.4 # $, cost for 1 gallon of fuel

annual_fuel = Cbhp * HF * Pcruise # lbs

annual_fuel_cost = annual_fuel / fuel_density * fuel_cost_gallon

print(f"Annual fuel cost: ${int(annual_fuel_cost):,}")

Annual fuel cost: $67,840


## Insurance 

The insurance cost for an airplane depends on various factors such as airplane model and pilot experience. Due to the large variability in the insurance premium, it is difficult to estimate insurance cost. In this section, a simple equation from Gudmundsson is used for an initial approximation which uses airplane's selling price (evaluated in unit cost section). A better estimate should be used, if available. Below code block computes insurance cost for the example airplane.

In [3]:
C_ac = 900000 # $
C_ins = 500 + 0.015 * C_ac # $/year, insurance

print(f"Insurance cost per year: ${int(C_ins):,}")

Insurance cost per year: $14,000


## Engine overhaul fund

The airplane engines typically require regular overhaul as recommended by the engine manufacturer. This overhaul cost can be estimated by assuming that this cost is amortized over the total flight hours between two overhauls. For Lycoming and Continental engines, it is found that $5 per flight hour is a prudent estimate for this overhaul cost. Below code block computes the annual engine overhaul cost for the example airplane:

In [4]:
N_engines = 2
C_overhaul = 5 * N_engines * HF # $/year

print(f"Engine overhaul cost per year: ${int(C_overhaul):,}")

Engine overhaul cost per year: $3,000


## Other costs

Apart from above mentioned costs, there are other expenses associated with operating an airplane. Firstly, the airplane needs to be stored at a main base which will incur some cost. The storage cost is set to \$250/month in this section. Secondly, there is an annual inspection cost for the airplane which is set to \$500. Lastly, if the airplane purchase is fully/partially funded by a financial institute, then there will be monthly loan payment. In this demonstration, it is assumed that there is no loan associated with airplane acquisition. Below code block computes these additional costs:

In [5]:
R_storage = 250 # $, storage cost per month
C_storage = 12 * R_storage # $, annual storage cost
C_inspection = 500 # $, inspection cost
C_loan = 0

print(f"Annual storage cost: ${C_storage:,}")
print(f"Annual inspection cost: ${C_inspection:,}")
print(f"Annual loan payment: ${C_loan:,}")

Annual storage cost: $3,000
Annual inspection cost: $500
Annual loan payment: $0


## Total annual cost

The total annual cost is simply the sum of the above costs. If required, other costs can be added as well. Below code block computes annual operating cost for the example airplane. It also computes the operating cost per flight hour.

In [6]:
C_year = C_maintenance + annual_fuel_cost + C_ins + C_overhaul + C_storage + C_inspection + C_loan # annual cost

C_hour = C_year / HF # cost per flight hour

print(f"Total yearly operating cost: ${int(C_year):,}")
print(f"Operating cost per flight hour: ${int(C_hour):,}")

Total yearly operating cost: $93,020
Operating cost per flight hour: $310


Below table summarizes the operating cost for the example airplane:

<div style="width:45%; margin: auto;">

Cost type | Annual Cost ($)
----- | :---:
Maintenance | 4,680
Fuel | 67,840
Insurance | 14,000
Engine overhaul fund | 3,000
Storage | 3,000
Annual inspection | 500
Loan | 0
**Total** | **93,020**
</div>

Based on the above table, it can be seen that the fuel is a major factor in the operating cost. Reducing the fuel consumption will lower the operating cost significantly. This concludes operating cost estimation for the example airplane.